apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -24,28 +26,24 @@ export function getAnimationLibraryFile(filename) {
   );
 }
 
-// Returns the list of default sprites in SpriteLab in English
```

Comment:
Since the tool to edit default sprites was built to 'almost' complete, we've finalized how the API is expected to work. The `getDefaultList`/`updateDefaultList` endpoints are outdated and replaced with `getDefaultListMetadata/updateDefaultListMetadata`. So here we are deleting the old endpoint and adding 'environment' specification to the new ones.